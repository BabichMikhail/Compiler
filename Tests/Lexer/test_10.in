program Test;
var 
	integer a;
	string b;
	Integer a;
begin
	integer a = 5;
	b = 'Hello world\n';
	c = 10;
end	