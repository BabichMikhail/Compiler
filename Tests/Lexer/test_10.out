1	1	program	program
1	9	identifier	Test
1	13	;	;
2	1	var	var
3	5	integer	integer
3	13	identifier	a
3	14	;	;
4	5	string	string
4	12	identifier	b
4	13	;	;
5	5	integer	Integer
5	13	identifier	a
5	14	;	;
6	1	begin	begin
7	5	integer	integer
7	13	identifier	a
7	15	=	=
7	17	integer_value	5
7	18	;	;
8	5	identifier	b
8	7	=	=
8	9	string_value	Hello world\n
8	24	;	;
9	5	identifier	c
9	7	=	=
9	9	integer_value	10
9	11	;	;
10	1	end	end
